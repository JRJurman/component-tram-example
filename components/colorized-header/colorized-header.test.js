const Tram = require('tram-one')
const html = Tram.html({
  'colorized-header': require('./colorized-header')['colorized-header']
})

describe('colorized-header', () => {
  it('should match snapshot', () => {
    const tree = html`<colorized-header color="red" />`
    expect(tree.outerHTML).toMatchSnapshot()
  })
})
