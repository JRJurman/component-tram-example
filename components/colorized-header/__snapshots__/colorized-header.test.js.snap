// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`colorized-header should match snapshot 1`] = `
"<h1 style=\\"
    color: red; background: black;
    cursor: pointer; text-align: center;
    padding: 1em; user-select: none;
    margin: -8px -8px 8px -8px;
  \\">
      test-app
    </h1>"
`;
